features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2145,6 +2145,7 @@
             </PreIssueAccessToken>
             <Authentication>
                 <Enable>{{actions.types.authentication.enable}}</Enable>
+                <DefaultUserStore>{{actions.types.authentication.default.userStore}}</DefaultUserStore>
```

Comment:
Do we want to break this as default.userStore ?
We won't have anything under default further

why not go with defaultUserStore as one key word